pipeline_games/conftest.py

```diff
@@ -96,3 +97,37 @@ def fake_response():
             'categories': [{'id': 2, 'description': 'Single-player'}],
             'genres': [{'id': '1', 'description': 'Action'}, {'id': '25', 'description': 'Adventure'}, {'id': '28', 'description': 'Simulation'}, {'id': '2', 'description': 'Strategy'}],
             'release_date': {'coming_soon': False, 'date': '5 Sep, 2023'}}
+
+
+@pytest.fixture
+def fake_raw_data():
+    return pd.DataFrame([[2246030, "Fake: Escape", "5 Sep, 2023", "Early Access,Clicker,Strategy", '£3.39', '£2.54',
+                          True, False, False, "Adventure,Early Access", "Fake, Fake2", "Fake"]], columns=['app_id', 'title',
+                        'release_date', 'user_tags', 'full price', 'sale price', 'windows', 'mac', 'linux', 'genres', 'developers', 'publishers'])
+
+
+@pytest.fixture
+def fake_data_with_tags():
+    return pd.DataFrame([[2246030, "Fake: Escape", "5 Sep, 2023", "Early Access,Clicker,Strategy", '£3.39', '£2.54',
+                          True, False, False, "Adventure,Early Access", "Fake", "Fake", "Early Access"]], columns=['app_id', 'title',
+                        'release_date', 'user_tags', 'full price', 'sale price', 'windows', 'mac', 'linux', 'genres', 'developers', 'publishers', 'tags'])
+
+
+@pytest.fixture
+def fake_date():
+    return "5 Sep, 2023"
+
+
+@pytest.fixture
+def fake_invalid_date():
+    return "30 Feb, 2023"
+
+
+@pytest.fixture
+def fake_price():
+    return "£5.30"
+
+
+@pytest.fixture
+def price_is_free():
+    return "Free to play"
```

Comment:
Are these needed as separate functions since they're not being used more than once? Unless you're planning on using them again